lookout/style/format/virtual_node.py

```diff
@@ -68,8 +68,8 @@ def __str__(self) -> str:
         return self.value
 
     def __repr__(self) -> str:
-        return ("VirtualNode(\"%s\", y=%s, start=%s, end=%s, node=%s, path=\"%s\")" % (
-                    self.value.replace("\n", "\\n").replace("\t", "\\t"),
+        return ("VirtualNode(%s, y=%s, start=%s, end=%s, node=%s, path=\"%s\")" % (
```

Comment:
repr should stay repr
```suggestion
        return ("VirtualNode(%s.decode(), y=%s, start=%s, end=%s, node=%s, path=\"%s\")" % (
```

Reply:
Alternatively, try `repr(self.value)`

Reply:
of course! `repr(self.value)` is a good one, thanks!